src/asset_folder_importer/premiere_get_referenced_media/processor.py

```diff
@@ -140,6 +140,17 @@ def process_premiere_fileref(filepath, server_path, vsproject, vs_pathmap=None,
     return item
 
 
+def update_metadata(vsproject, invalid_media_paths):
+    lg.debug("update_metadata: called with {0}".format(invalid_media_paths))
+    current_value_of_field = vsproject.get('gnm_project_invalid_media_paths', allowArray=True)
+    lg.info("current_value_of_field: {0}".format(current_value_of_field))
```

Comment:
update these to `debug` and prefix with the function name before deployment pls

Reply:
Will do.